lib/swaps/Swaps.ts

```diff
@@ -502,19 +479,13 @@ class Swaps extends EventEmitter {
       }
     }
 
-    let cmdLnd: LndClient;
-    // running as taker
-    switch (deal.makerCurrency) {
-      case 'BTC':
-        cmdLnd =  this.lndBtcClient;
-        break;
-      case 'LTC':
-        cmdLnd = this.lndLtcClient;
-        break;
-      default:
-        // Can't be if we check that pairID is LTC/BTC only (for now). Still...
-        return;
+    const cmdLnd = this.lndClients[deal.makerCurrency];
+    if (!cmdLnd) {
+      // Can't be if we check that pairID is LTC/BTC only (for now). Still...
```

Comment:
Perhaps change this comment to explain that we already checked this when swap was initiated.